cmd/minify/main.go

```diff
@@ -620,7 +628,7 @@ func createTasks(inputs []string, output string) ([]Task, []string, error) {
 				}
 				return nil
 			}
-			if err := WalkDir(DirFS("."), input, walkFn); err != nil {
+			if err := WalkDir(DirFS(input), ".", walkFn); err != nil {
```

Comment:
I think keeping this as it was will prevent the changes above for joining `root` with `path`. I don't think that the `walkFn` will ever have an absolute path?

Reply:
In my testing, `walkFn()` _was_ always called with a relative path - but I'm taking a conservative approach, concious that minify can be invoked in a number of different ways with different paramters.

I didn't want to introduce a bug by prepending the path with `root` if it wasn't needed, so I wrote the code defensively.

I can remove the `filepath.IsAbs()` test (line 588 of `main.go`) if you prefer.

Reply:
You were right, this was not working well with absolute input paths, thanks!